### ISSUE TITLE
Magnifier shakes when typing into input fields

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#115035]
Refs: https://github.com/microsoft/vscode/issues/112490

Windows, turn on Windows Magnifier, make sure you have the Magnifier View Docked (under magnifier settings)

1. VS Code, press F1 (opens the quick pick), start typing, notice how the docked magnifier shakes left and right making it very hard to see what is going on while typing
2. Now type in the editor, notice how the magnifier shakes considarably less

This works well with the ZoomText magnifier software. So there might be an issue with the Winodws Magnifier. I am trying to connect with the Magnifier team and will let you know if I have updates.

Might be the same underlying issue https://github.com/microsoft/vscode/issues/115021

**Comment by isidorn** [comment]
@Tyriar was not able to reproduce a similar issue in the Terminal.

Let me know if you can  reproduce. It might be due to my VM...

**Comment by bpasero** [comment]
I can reproduce but I also see shaking (less though) in other Electron apps (such as Fiddle). I would suggest to file a bug against Electron or Windows or even Chrome since I don't think this is something I can fix by changing quick pick code.

**Comment by isidorn** [comment]
Thanks @bpasero for trying with other Electron app. I have emailed the Windows Magnifier team, let's see if they have ideas.

**Comment by isidorn** [comment]
This should be better with the latest VS Code release.
